src/pages/AllRestaurantsDetails/components/Header.tsx

```diff
@@ -0,0 +1,8 @@
+import styled from '@emotion/styled'
+import React from 'react'
+
+const StyledHeader = styled.div``
```

Comment:
No estamos usando el StyledHeader! Se queja el linter

```suggestion
import React from 'react'

```